src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -224,6 +234,12 @@ static CliRootCommand CreateCommandLineParser()
         Required = false
     };
 
+    var serverPipeNameOption = new CliOption<string?>("--pipe")
+    {
+        Description = "The name of the pipe used to connect to the server.",
```

Comment:
This is the server, isn't it? Shouldn't this be "The name of the pipe the server will connect to"

Reply:
I was thinking, if you read this then you read it in the console, probably because you want to use the server to connect as a client to it.

But I can for sure change it, what would you suggest?

Reply:
"The name of the pipe the server will connect to."

seems good to me.